deploy.cake

```diff
@@ -174,13 +175,59 @@ Task("Publish-VstsTask")
     }
 });
 
+
+Task("Publish-DockerImage")
+    .IsDependentOn("DownloadGitHubReleaseArtifacts")
+    .Does(() =>
+{
+    var returnCode = StartProcess("docker", new ProcessSettings
```

Comment:
Any reason for not using the Cake.Docker addin for this?

https://github.com/MihaMarkic/Cake.Docker